nym-vpn-app/src-tauri/src/commands/account.rs

```diff
@@ -65,7 +65,7 @@ pub async fn is_account_stored(grpc: State<'_, GrpcClient>) -> Result<bool, Back
     grpc.is_account_stored()
         .await
         .map_err(|e| {
-            error!("failed to check stored account: {e}");
+            debug!("failed to check stored account: {e}");
```

Comment:
Why this level downgrade? As I see it we want such critical error to be captured as error.
Here if daemon is failing to check for account stored.